### ISSUE TITLE
Confusing project name?

### ISSUE THREAD

**Issue by fractalf** [pipewire-debian/pipewire-debian#52]
Why is this called pipewire-**debian** when it is clearly just targeted for ubuntu?

**Comment by souravdas142** [comment]
Thanks for writing.
Of course its targeted for ubuntu but you know Its not  targeted only and only ubuntu. The project targets ubuntu (and its  derived distros (Mint, Elementary, **Pop_Os** etc.)), Debian (and its derived distros i.e. Kali, Parrot, MX Linux, etc. etc.). And after all whatever it targets the package architecture totally depends on Debian right ? So ubuntu is subset of Debian right ? And as this package targets Debian based distros it automatically refer to ubuntu itself.

In a nutshell I am targeting whole Debian family can be installed from this. Now you can tell I am cocktailing things over Debian based system right ? But no, you know all packages (and its subpackages) are completely independent from the debian packages. There is no harm to install from here.

For more see [Readme](../README.md), You can find some debian based instruction also there. 

So what is your point of view regarding this you can mention also? Thank you.

**Comment by souravdas142** [comment]
Hii @fractalf ,

Would you like to give any suggestion or you are convinced with above answer ?

**Comment by fractalf** [comment]
Hi, well if the target is ubuntu (and deriveties) I would call it **pipewire-ubuntu** ;)

I was looking for something to make pipewire work with latest debian (stable).

**Comment by danielecr** [comment]
doing:

> sudo add-apt-repository ppa:pipewire-debian/pipewire-upstream

does not work, it gives error
